index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="data.js"></script>
+  <script src="db.json"></script>
```

Comment:
The `db.json` file doesn't need to be added here; the only place we need to use that file is with `json-server` (technically, it's "backend code").